src/api/app/assets/javascripts/webui2/collapsible_text.js

```diff
@@ -1,9 +1,12 @@
 $(document).ready(function() {
   $('.obs-collapsible-textbox').on('click', function() {
```

Comment:
To handle double-clicks, this line could be:

`$('.obs-collapsible-textbox').on('click dblclick', function() {`